samples/Sentry.Samples.Console.Basic/Program.cs

```diff
@@ -1,13 +1,91 @@
+/*
+ * This sample demonstrates the following basic features of Sentry, via a .NET console application:
+ * - Error Monitoring (both handled and unhandled exceptions)
+ * - Performance Tracing (Transactions / Spans)
+ * - Release Health (Sessions)
+ * - MSBuild integration for Source Context (see the csproj)
+ *
+ * For more advanced features of the SDK, see Sentry.Samples.Console.Customized.
+ */
+
 using Sentry;
 
-using var _ = SentrySdk.Init(o =>
+// Initialize the Sentry SDK, disposing when the application is shutting down.
+using var _ = SentrySdk.Init(options =>
 {
-    // The DSN is required.
-    o.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537";
+    // A Sentry Data Source Name (DSN) is required.
+    // See https://docs.sentry.io/product/sentry-basics/dsn-explainer/
+    // You can set it in the SENTRY_DSN environment variable, or you can set it in code here.
+    // options.Dsn = "... Your DSN ...";
 
     // When debug is enabled, the Sentry client will emit detailed debugging information to the console.
-    o.Debug = true;
+    // This might be helpful, or might interfere with the normal operation of your application.
+    // We enable it here for demonstration purposes.
+    // You should not do this in your applications unless you are troubleshooting issues with Sentry.
+    options.Debug = true;
+
+    // This option is recommended, which enables Sentry's "Release Health" feature.
+    options.AutoSessionTracking = true;
+
+    // This option is recommended for client applications only. It ensures all threads use the same global scope.
+    // If you are writing a background service of any kind, you should remove this.
+    options.IsGlobalModeEnabled = true;
+
+    // This option will enable Sentry's tracing features. You still need to start transactions and spans.
+    options.EnableTracing = true;
 });
 
-// The following unhandled exception will be captured and sent to Sentry.
-throw new Exception("test");
+// This starts a new transaction and attaches it to the scope.
+var transaction = SentrySdk.StartTransaction("Program Main", "function");
+SentrySdk.ConfigureScope(scope => scope.Transaction = transaction);
+
+// Do some work. (This is where you'd have your own application logic.)
+FirstFunction();
+SecondFunction();
+ThirdFunction();
+
+// Always try to finish the transaction successfully.
+// Unhandled exceptions will fail the transaction automatically.
+// Optionally, you can try/catch the exception, and call transaction.Finish(exception) on failure.
+transaction.Finish();
+
+void FirstFunction()
+{
+    // This shows how you might instrument a particular function.
+    var span = transaction.StartChild("function", nameof(FirstFunction));
+
+    // Simulate doing some work
+    Thread.Sleep(100);
```

Comment:
Most of the code these days is async. Should we turn these functions into asynchronous and use `await Task.Delay(100)` instead?